content/_partials/thank-you-downloading-windows-installer.html.haml

```diff
@@ -8,6 +8,30 @@
 
   %a#startDownloadLink{:href => page.release_url}
     Click this link
+    
+%h2
+  Changing boot configuration
+
+%p
+  By default, your Jenkins runs at 
+  %a{:href => 'http://localhost:8080/'}
+    https://localhost:8080/. 
+  This can be changed by editing 
+  %code
+    jenkins.xml
+  , which is located in your installation directory. This file is also the place to change other boot configuration parameters, such as JVM options, HTTPS setup, etc.
+    
+%h2 
+  Starting/stopping the service
+  
+%p
+  Jenkins is installed as a Windows service, and it is configured to start automatically upon boot. To start/stop them manually, use the service manager from the control panel, or the sc command line tool.
+  
+%h2
+  Inheriting your existing Hudson/Jenkins installation
```

Comment:
Not sure if we should still mention Hudson @MarkEWaite @oleg-nenashev? But I would at least change  Hudson/Jenkins to Jenkins/Hudson so that the title and paragraph match. 

```suggestion
  Inheriting your existing Jenkins/Hudson installation
```

Reply:
No Hudson mentions please

Reply:
```suggestion
  Inheriting your existing Jenkins installation
```